README.md

```diff
@@ -62,7 +62,7 @@ routerConfig : TransitRouter.Config Route Action Model
 routerConfig :
   { mountRoute : Route -> Route -> Model -> (Model, Effects Action)
   , getDurations : Route -> Route -> Model -> (Float, Float)
-  , actionWrapper : TransitRouter.Action -> Action
+  , actionWrapper : (TransitRouter.Action, Route) -> Action
```

Comment:
Thanks for the PR! 
You left a comma here, also parenthesis are not necessary: `actionWrapper : TransitRouter.Action Route -> Action`